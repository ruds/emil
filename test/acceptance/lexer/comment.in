(**)
foo
(* *)
foo
(* foo *)
foo
(* (* foo *)
foo
*)
foo
R"(**)"
"(*" (* *) foo
(* "*)
foo
(*) *)
foo
*)
(*xx*)
(* (**)
