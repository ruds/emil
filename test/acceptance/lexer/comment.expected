0002 ID_WORD 666F6F foo
0004 ID_WORD 666F6F foo
0006 ID_WORD 666F6F foo
0010 ID_WORD 666F6F foo
0011 STRING 2A2A R"(**)"
0012 STRING 282A "(*"
0012 ID_WORD 666F6F foo
0014 ID_WORD 666F6F foo
0016 ID_WORD 666F6F foo
0017 ERROR
0019 ERROR
0020 END_OF_FILE
